### PR TITLE
What is limiting our ability to predict network structure?

### PR DESCRIPTION
**Pull request by TanyaS08**
added placeholder text

### CODE REVIEW

File: manuscript.md [hunk 1]
```diff
@@ -12,6 +16,30 @@
 
 ## What is currently limiting our ability to predict network structure?
 
+* As this will probably play a part in up setting the tone/concept for the ms
```
Comment (tpoisot) [inline manuscript.md:19]: This currently lacks a bit of referencing - you can have a look at the preprint I shared in the github teams, and in particular the Jordano papers from 2016

File: manuscript.md [hunk 2]
```diff
@@ -12,6 +16,30 @@
 
 ## What is currently limiting our ability to predict network structure?
 
+* As this will probably play a part in up setting the tone/concept for the ms 
+I guess it makes sense to make sure we address the need for ‘tools’ (as 
+granted by the all seeing light of ML) That being said might be nice to bring 
+in a general point for each section - if at all possible...
+
+The scarcity of data on species interactions is certainly one of the biggest
```
Comment (tpoisot) [inline manuscript.md:24]: Needs a sentence explaining where the scarcity is from: (i) detecting species, (ii) detecting species together, (iii) detecting species interactions

File: manuscript.md [hunk 3]
```diff
@@ -12,6 +16,30 @@
 
 ## What is currently limiting our ability to predict network structure?
 
+* As this will probably play a part in up setting the tone/concept for the ms 
+I guess it makes sense to make sure we address the need for ‘tools’ (as 
+granted by the all seeing light of ML) That being said might be nice to bring 
+in a general point for each section - if at all possible...
+
+The scarcity of data on species interactions is certainly one of the biggest 
+barriers in predicting network structure. Machine learning methods, especially deep
```
Comment (tpoisot) [inline manuscript.md:25]: You can cite de Aguiar et al about how the interaction sampling issues can lead to biased network structure estimates

File: manuscript.md [hunk 4]
```diff
@@ -12,6 +16,30 @@
...
+
+The scarcity of data on species interactions is certainly one of the biggest 
+barriers in predicting network structure. Machine learning methods, especially deep 
+learning methods, need a lot of data. Compounding the scarcity of data the 
+data that are available are biased toward food webs and select mutualistic 
+networks, and could prove to be a limiting factor when trying to understand or
+predict networks of *other* interaction types. A better understanding of how 
+different types of networks differ from each another would serve as a starting 
+point, although this in itself could be a potential hurdle 
+[@Michalska-Smith2019TelEco].
```
Comment (tpoisot) [inline manuscript.md:32]: this is a good point but under-developed - why is it important, and how is it going to help? If it's a starting point, what is next?

File: manuscript.md [hunk 5]
```diff
@@ -12,6 +16,30 @@
...
+
+The scale at which we are able to predict networks are currently limited to
```
Comment (tpoisot) [inline manuscript.md:34]: What scale? Space? Taxonomy? Clarify.

File: manuscript.md [hunk 6]
```diff
@@ -12,6 +16,30 @@
...
+networks, and could prove to be a limiting factor when trying to understand or
+predict networks of *other* interaction types. A better understanding of how 
+different types of networks differ from each another would serve as a starting 
+point, although this in itself could be a potential hurdle 
+[@Michalska-Smith2019TelEco].
+
+The scale at which we are able to predict networks are currently limited to 
+the resolution of the data available. Regarding levels of organisation our 
+understanding of individual based networks is still in its infancy 
+[@Guimaraes2020StrEco] and acts as a 'lower-limit' at which we
```
Comment (tpoisot) [inline manuscript.md:37]: I would cite a bunch of individual-based networks papers here

File: manuscript.md [hunk 7]
```diff
@@ -12,6 +16,30 @@
...
+point, although this in itself could be a potential hurdle 
+[@Michalska-Smith2019TelEco].
+
+The scale at which we are able to predict networks are currently limited to 
+the resolution of the data available. Regarding levels of organisation our 
+understanding of individual based networks is still in its infancy 
+[@Guimaraes2020StrEco] and acts as a 'lower-limit' at which we 
+would be able to predict networks. On the note of scale, the resolution of 
+environmental (or landscape) data would also limit our ability to predict 
+networks at finer scales, although current trends in e.g. remote sensing would
```
Comment (tpoisot) [inline manuscript.md:40]: https://www.frontiersin.org/articles/10.3389/fenvs.2019.00197/full and the papers cited therein about interactions should be cited here

**Review by tpoisot — CHANGES_REQUESTED** [review]

**Comment by tpoisot** [comment]
I have nuked the bibliography file, please re-export

**Comment by TanyaS08** [comment]
I have an admin question for @tpoisot - did you 'push' what was in the integration branch for section one into this branch as well but the Section 1 integration branch still exists and when this is a bit more fleshed out it would get merged with that branch? It's late on a Tuesday evening and everything is just a little bit fuzzy...

**Comment by TanyaS08** [comment]
I think we're beginning to take the steps from _it exists_ to _its getting functional_. There are still a few points that need to be raised/refined and I'm sure @FrancisBanville will have thoughts™ he's like to add

**Review by tpoisot — APPROVED** [review]

**Review by FrancisBanville — APPROVED** [review]
I think it is ready to be integrated @TanyaS08?

**Pull request merged**